Project.toml

```diff
@@ -9,8 +9,11 @@ DSP = "717857b8-e6f2-59f4-9121-6e50c889abd2"
 DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 DocStringExtensions = "ffbed154-4ef7-542d-bbb7-c09d3a79fcae"
+GR = "28b8d3ca-fb5f-59d9-8090-bfdbd6d07a71"
+JLD2 = "033835bb-8acc-5ee8-8aae-3f567f8a3819"
 JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
```

Comment:
We have a `Project.toml` for `src/`, a `Project.toml` for the `test/` and one for each example. The reason for this is so items that aren't used in `src/` such as `Plots` and `JLD2` etc. aren't loaded, but they will be when e.g. you run an example etc. 

Please move these dependencies to the relevant `Project.toml`
PS. I think this is also the cause of the failed unit test.

Reply:
(PS GR is usually loaded with `Plots.jl` that I knew)